great-idea/css/index.css

```diff
@@ -63,4 +63,101 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.logo {
+    display: inline-block;
```

Comment:
For the MVP of this assignment, you are meant to refactor all inline-block elements to use Flexbox instead. For example, you can put `display: flex;` on `div.main-header`, then use properties like `justify-content: space-between;` to separate the navigation bar and the logo.